### PR TITLE
improve mntr regex to match user specific keys.

### PR DESCRIPTION
**Pull request by powerLambda**
### Required for all PRs:

Found zookeeper input plugin error as below:

> [inputs.zookeeper] Error in plugin: unexpected line in mntr response: "zk_avg_pull-daily-attendence-data-testByQA_read_per_namespace\t137.1803"

This PR is to improve mntr regex to match user specific keys.

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by powerLambda** [comment]
Bug link #7550

File: plugins/inputs/zookeeper/zookeeper.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-var zookeeperFormatRE = regexp.MustCompile(`^zk_(\w+)\s+([\w\.\-]+)`)
+var zookeeperFormatRE = regexp.MustCompile(`^zk_([\w\.\-]+)\s+([\w\.\-]+)`)
```
Comment (srebhan) [inline plugins/inputs/zookeeper/zookeeper.go:20]: I think this is not correct as you now also accept items with "." or "-" only. I guess the correct regex should be
```suggestion
var zookeeperFormatRE = regexp.MustCompile(`^zk_(\w[\w\.\-]*)\s+([\w\.\-]+)`)
```

**Review by srebhan — CHANGES_REQUESTED** [review]

**Comment by srebhan** [comment]
@powerLambda can you please check the comment so we can go forward with this PR!?

**Comment by srebhan** [comment]
@powerLambda ping!

**Comment by sjwang90** [comment]
@powerLambda - can you please let us know if you can make the changes to @srebhan's comment. Otherwise we can look for someone on the influx side to finish it up.

**Comment by powerLambda** [comment]
@sjwang90 sorry for the late reply, crazy busy for a while.

**Review by srebhan — APPROVED** [review]
Looks good to me.

**Review by ssoroka — APPROVED** [review]

**Pull request merged**